### DISCUSSION TITLE
Cropping: Smart cropping cut-off

### DISCUSSION THREAD

**Discussion by curquiza** [meilisearch/product#123] [Feedback & Feature Proposal]
Currently, the crop is done by char, and does not cut a word in the middle, which is **really** convenient 🙂

But would it be possible to remove the punctuation char (and the following space of course) that are at the beginning of the cropped string?

<img width="790" alt="Capture d’écran 2020-08-20 à 09 07 52" src="https://user-images.githubusercontent.com/20380692/90728344-0ba4b600-e2c5-11ea-9015-e59d3e13749f.png">

⚠️ The punctuation char should still be present if there are at the end of the crop value.

Not an emergency feature but it would definitively make MeiliSearch most enjoyable to use in front-end 😁

**Comment by gmourier** [comment]
It would be great to have this!
